### PR TITLE
Feat/zoom bubbles

### PR DESCRIPTION
**Pull request by FarisElghlan**
Semantic. Zooming.

### CODE REVIEW

**Comment by wouterjsmit** [comment]
You forgot a few labels.

**Comment by cbilstra** [comment]
Can we split the work somehow? @FarisElghlan @cathmer maybe to can make a list of important files to review and then @justinvdk @Pathemeous and me can split the review work?

File: PL2/PL2-core/pom.xml [hunk 1]
```diff
@@ -26,6 +26,11 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-all</artifactId>
+			<version>1.9.5</version>
+		</dependency>
+        <dependency>
```
Comment (justinvdk) [inline PL2/PL2-core/pom.xml:33]: Please use spaces as indentation

**Comment by wouterjsmit** [comment]
I suggest that to speed up this process, we do the following.
- Let Faris check the last changes (hotfixes) and review those.
- @cbilstra, @justinvdk and I review as much as possible of this.
- We create new issues for large problems that can't be fixed straight away.
- We merge this ASAP

**Comment by FarisElghlan** [comment]
Hotfixes look good.

**Pull request merged**